stdlib/experimental/usage/usage.flux

```diff
@@ -12,7 +12,13 @@ import "http"
 // return raw usage data rather than the downsampled data returned by default.
 // Note that unlike the range function, the stop argument is required here,
 // pending implementation of https://github.com/influxdata/flux/issues/3629.
-from = (start, stop, host="", orgID="", token="", raw=false) => {
+from = (
+        start,
+        stop,
+        host="",
+        orgID="",
+        token="",
+        raw=false) => {
```

Comment:
This is just a personal preference, but in this case I would expect the right paren and everything after it to get moved to the next line, and for a comma to be appended to line 21. Maybe that's outside of the scope of this PR though. Something to bring up with the team perhaps?

```suggestion
from = (
        start,
        stop,
        host="",
        orgID="",
        token="",
        raw=false,
) => {
```

Reply:
@nathanielc I don't know if we should go through the effort to implement this, or if we should just file a separate issue and let this PR merge. Thoughts?

Reply:
This would be a very easy (1 line) logic fix. We should just ask the team what they prefer and I can push the change if needed.

Reply:
Yes lets change this to add a newline after the last parameter. The  precedent in Flux has been to make all lines in a list the same including the last line, So for example arrays and records format with the last element and key/value pair on their own line and the closing delimiter on its own line. 

```js
a = [
   1,
   2.
   3,
]
b = {
    a: 1,
    b: 2,
    c: 3,
}
````

Reply:
Ok sounds good.